src/move/econia/Move.toml

```diff
@@ -4,7 +4,7 @@ version = "4.1.0"             # Per SemVer.
 upgrade_policy = "compatible"
 
 [addresses]
-econia = "0xc0deb00c405f84c85dc13442e305df75d1288100cdd82675695f6148c7ece51c"
+econia = "_"
```

Comment:
This should be left as the mainnet contract address

If someone needs to update to `_` for publishing a local copy etc. then they can do it temporarily/manually

Reply:
This was left over from when I was trying to do automatic deployments -- it will be necessary in the future if we do end up doing deployments with scripts and such. Will leave it out of this diff though as it isn't strictly necessary.